alternator/auth.cc

```diff
@@ -31,11 +32,12 @@ future<std::string> get_key_from_roles(service::storage_proxy& proxy, auth::serv
     dht::partition_range_vector partition_ranges{dht::partition_range(dht::decorate_key(*schema, pk))};
     std::vector<query::clustering_range> bounds{query::clustering_range::make_open_ended_both_sides()};
     const column_definition* salted_hash_col = schema->get_column_definition(bytes("salted_hash"));
-    if (!salted_hash_col) {
+    const column_definition* can_login_col = schema->get_column_definition(bytes("can_login"));
```

Comment:
would be better to use role_manager.can_login(), but I understand it's one extra query, so for now we probably can't afford it, but it will have to be eventually refactored to use the same interfaces

Reply:
I suggest to create an Issue about it or maybe @nyh can add a `// TODO: xxx` comment in the source code

Reply:
I don't understand what you guys are suggesting. This is pre-existing code which already had a query and I am just looking at a second column. Whether or not this pre-existing code was the right or wrong way to do this query, I don't know :-(

Reply:
it was wrong to not use auth interfaces but it's pre-existing and we can't improve w/o unified cache so for now I suggest to do nothing